ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -291,7 +246,9 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
                                          subtitleTrailingAccessoryView: subtitleTrailingAccessoryView,
                                          footerLeadingAccessoryView: footerLeadingAccessoryView,
                                          footerTrailingAccessoryView: footerTrailingAccessoryView)
-        customViewSize.validateLayoutTypeForHeightCalculation(&layoutType)
+        if customViewSize == .medium && layoutType == .oneLine {
+            layoutType = .twoLines
```

Comment:
Can you leave some comments explaining this logic? Seems odd to me that there's a check that says `if layoutType is oneLine then make it twoLines` without context.